packages/xrpl/src/Wallet/defaultFaucets.ts

```diff
@@ -29,21 +34,19 @@ export const FaucetNetworkPaths: Record<string, string> = {
  * @throws When the client url is not on altnet or devnet.
  */
 export function getFaucetHost(client: Client): FaucetNetwork | undefined {
-  const connectionUrl = client.url
-
-  // 'altnet' for Ripple Testnet server and 'testnet' for XRPL Labs Testnet server
-  if (connectionUrl.includes('altnet') || connectionUrl.includes('testnet')) {
-    return FaucetNetwork.Testnet
-  }
-
-  if (connectionUrl.includes('sidechain-net2')) {
```

Comment:
We need to mention that support for `sidechain-net2` is being deprecated in the ReleaseNotes. Has this sidechain been down for long enough time?